src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java

```diff
@@ -22,90 +22,46 @@ public static String setResult(Integer myNumber, String result){
     }
 
 
-    @GetMapping("/insertValue")
-    public static void addValues(@RequestParam(value = "id") Long id, @RequestParam (value = "myNumber") Integer myNumber, @RequestParam (value = "name") String name) {
-
+    public static void connectionAndExecute(String query, Integer myNumber){
```

Comment:
Do we need `myNumber` in this method?
It seems like it is not used and we can just remove it and also we won't need to execute this method with 0s where myNumber is not needed